ansible/vars/production/production_public.yml

```diff
@@ -204,6 +204,7 @@ localsettings:
   MAPS_LAYERS: "{{ localsettings_private.MAPS_LAYERS }}"
   MIA_THE_DEPLOY_BOT_API: "{{ localsettings_private.MIA_THE_DEPLOY_BOT_API }}"
   MOBILE_INTEGRATION_TEST_TOKEN: "{{ localsettings_private.MOBILE_INTEGRATION_TEST_TOKEN }}"
+  ONBOARDING_DOMAIN_TEST_DATE: (2016, 12, 8)
```

Comment:
just checking that this will be a python string, not a tuple when it's put in the localsettings